tests/serialise/xml-serialise/ietf-xml-serialise.yang

```diff
@@ -175,6 +175,12 @@ module ietf-xml-serialise {
         }
       }
 
+      leaf missing-identityref {
```

Comment:
`s/missing/unset/` -- "missing" sounds like the identityref doesn't have a valid base.